packages/components/src/panel/row.tsx

```diff
@@ -0,0 +1,37 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { forwardRef } from '@wordpress/element';
+import type { ForwardedRef } from 'react';
+
+/**
+ * Internal dependencies
+ */
+import type { PanelRowProps } from './types';
+
+function UnforwardedPanelRow(
+	{ className, children }: PanelRowProps,
+	ref: ForwardedRef< HTMLDivElement >
+) {
+	return (
+		<div
+			className={ classnames( 'components-panel__row', className ) }
+			ref={ ref }
+		>
+			{ children }
+		</div>
+	);
+}
+
+/**
+ * `PanelRow` is a generic container for rows within a `PanelBody`.
+ * It is a flex container with a top margin for spacing.
```

Comment:
I understand that these info can be useful for a dev using this component, but should we expose implementation details like layout APIs (flex) and design details (margin top) ? An interesting dilemma, I personally don't have a strong opinion

Reply:
For this one I decided it was necessary to mention because it's essential to what PanelRow is. Those two styles are the only reason the component exists, and I phrased it in a way that we can stick to as a contract. In other words, if it ceases to be a flex container, or if it ceases to have a top margin, I would consider it a breaking change.